model_cards/seyonec/ChemBERTa-zinc-base-v1/README.md

```diff
@@ -0,0 +1,14 @@
+# RoBERTA: Training a BERT-like transformer model for masked language modelling of chemical SMILES strings.
+
+Deep learning for chemistry and materials science remains a novel field with lots of potiential. However, the popularity of transfer learning based methods in areas such as NLP and computer vision have not yet been effectively developed in computational chemistry + machine learning. Using HuggingFace's suite of models and the ByteLeve tokenizer, we are able to train on a large corpus of 100k SMILES strings from a commonly known benchmark dataset, ZINC.
+
+Training RoBERTa over 5 epochs, the model achieves a decent loss of 0.398, but may likely continue to decline if trained for a larger number of epochs. The model can predict tokens within a SMILES sequence/molecule, allowing for variants of a molecule within discoverable chemical space to be predicted.
```

Comment:
Is there a leaderboard of losses on ZINC? (on https://zinc.docking.org/?) that maybe we can link to here?

Reply:
This is a great question. Unfortunately, to the best of my knowledge, there doesn't seem to be a global leaderboard of losses similar to CV + ImageNet for ZINC (or other public SMILES datasets). Hopefully, this changes soon! The closest I've seen is GuacaMol (https://benevolent.ai/blog/guacamol-a-new-framework-to-compare-models-for-de-novo-molecular-design), which doesn't apply to MLM.

Reply:
The paper I am working on to reproduce: https://arxiv.org/pdf/1908.06760.pdf about Molecule Transformers and Drug Target Interaction (MT-DTI) use ```Kiva``` and ```Davis``` datasets. You can see it in the top of page 11

Reply:
You have that model in the hub @julien-c (https://huggingface.co/mrm8488/chEMBL_smiles_v1) based on the paper https://onlinelibrary.wiley.com/doi/full/10.1002/minf.201700111
https://twitter.com/mrm8488/status/1242089078499262464
Sorry for gif bad quality